pywbemtools/pywbemcli/_connection_repository.py

```diff
@@ -56,11 +56,77 @@
 BAK_FILE_SUFFIX = 'bak'
 
 
-class ConnectionsFileNotFoundError(Exception):
+class ConnectionsFileError(Exception):
     """
     Exception indicating that the connections file was not found.
```

Comment:
The description is no longer true, because for that case there is now `ConnectionsFileNotFoundError`.

Also, is this meant to be an exception base class that is never actually raised? If so, state it.